src/Sylius/Behat/Context/Ui/Admin/ResettingPasswordContext.php

```diff
@@ -25,10 +25,10 @@
 final class ResettingPasswordContext implements Context
 {
     public function __construct(
+        private NotificationCheckerInterface $notificationChecker,
         private RequestPasswordResetPage $requestPasswordResetPage,
-        private ResetPasswordPageInterface $resetPasswordPage,
         private ResetElementInterface $resetElement,
-        private NotificationCheckerInterface $notificationChecker,
+        private ResetPasswordPageInterface $resetPasswordPage,
```

Comment:
Why the shuffling?

Reply:
Alphabetical order 💃

Reply:
IMO revert this order change, we are not using the alphabetical order in constructors and I would be for not changing the existing constructor even if it is only the behat context.